<launch>
    <node pkg="velodyne_detect_person" type="set_static_background" name="set_static_background"/>
	<node pkg="velodyne_detect_person" type="extract_clusters" name="extract_clusters" output="screen" cwd="ROS_HOME"/>
	<node pkg="velodyne_detect_person" type="tf_publisher" name="tf_publisher" />
	<node pkg="velodyne_detect_person" type="find_person" name="find_person" output="screen"/>

	<node pkg="velodyne_detect_person" type="robot_data" name="robot_data"/>
</launch>
